src/documents/index.html.md.handlebars

```diff
@@ -2,41 +2,33 @@
 title: Welcome to The Inclusive Design Guide
 layout: index-template
 ---
-The guide is organised into four sections:
 
-<div id="idg-index-icon-list">
+The Inclusive Design Guide can be applied to digital design as well as to the design of **services**, the **built environment** and **physical products**. It can be applied to processes like **workshops**, **meetings**, **conferences**, and even our daily interactions with one another. It can be used by anyone. The Guide is ever-evolving as we learn more about how to design inclusion into all that we do. **We rely on your feedback and contributions to continue developing these ideas**.
+
+
+<div id="idg-index-dimensions-list">
```

Comment:
Re-use the existing style "idg-highlight-section" which is similar. Remove idg-index-dimensions-list as necessary.